src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -55,7 +55,7 @@ public class InstancesController : Controller
             _applicationRepository = applicationRepository;
             this.logger = logger;
             this.bridgeRegistryClient = bridgeClient;
-            this.bridgeRegistryClient.BaseAddress = new Uri(generalSettings.Value.GetBridgeRegisterApiEndpoint());
+            this.bridgeRegistryClient.BaseAddress = new Uri(generalSettings.Value.GetBridgeRegisterApiEndpoint() ?? "http://bridge/");
```

Comment:
is there ever a case where the base address is set to http:/bridge ?

Reply:
I have reworked logic and removed bridge